README.md

```diff
@@ -133,13 +133,23 @@ In this mode, the `Makefile` will fetch secrets using `aws secretsmanager` and `
 
 ### Tests
 
-To run the unit tests
+To run the unit tests for the PHP applications:
 
 ```bash
 make dc-unit-tests
 ```
 
-For how to run the functional tests, please see seperate README in tests/functional directory
+To run the unit tests for the performance platform (Python):
+
+```zsh
```

Comment:
could this be another make target? ok if not...

Reply:
Could be, but would need to put in a docker container to do that. It could potentially be piggybacked on perfplatconfig, which only runs locally and configures perfplat for the local environment. For example, we could run the docker container, mount the volume and run the unit tests from within it. What do you reckon?